legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/resources/VersionsResource.java

```diff
@@ -140,6 +144,10 @@ public Version createVersion(@PathParam("projectId") String projectId, CreateVer
         {
             throw new LegendSDLCServerException("Server does not support creating project version(s)", Response.Status.METHOD_NOT_ALLOWED);
         }
+        if (projectApi.getProject(projectId).getProjectType() == ProjectType.EMBEDDED)
+        {
+            throw new LegendSDLCServerException("Cannot release project of " + projectApi.getProject(projectId).getProjectType() + " type", Response.Status.CONFLICT);
+        }
```

Comment:
The error message should be in terms of creating a version, not of releasing a project. And it's best to just get the project type once since each call potentially makes a remote call.
```suggestion
        ProjectType type = projectApi.getProject(projectId).getProjectType();
        if (type == ProjectType.EMBEDDED)
        {
            throw new LegendSDLCServerException("Creating a version of a project of type " + type + " is not allowed", Response.Status.CONFLICT);
        }
```

Reply:
done, using 1 call - however to ProjectConfigurationApi as getProjectType on ProjectApi returns null